frontend/src/AuthRoute.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Route, Redirect } from 'react-router-dom';
+
+const propTypes = {
+  isLoggedIn: PropTypes.bool.isRequired,
+  component: PropTypes.object,
+  redirect: PropTypes.string,
+  render: PropTypes.func,
+};
+const defaultProps = {
+  component: null,
+  redirect: '',
```

Comment:
I suggest to make redirect default to `"/login"`, that way you can get rid of all `redirect="/login"` in `App.js` and still have an option to set custom redirect route.

Reply:
that's a good point.

Reply:
i have nothing to add, approved